crates/dreammaker/src/config.rs

```diff
@@ -86,10 +86,12 @@ pub enum WarningLevel {
 
 /// Available debug engines.
 #[derive(Debug, Deserialize, Clone, Copy, PartialEq)]
+#[derive(Default)]
```

Comment:
```suggestion
#[derive(Debug, Default, Deserialize, Clone, Copy, PartialEq)]
```
actually this should just be on one line lol